### ISSUE TITLE
New update turns Vesktop to solid white, completely unusable

### ISSUE THREAD

**Issue by HasuMigu** [Vencord/Vesktop#125]
OS: Arch Linux 
DE: KDE Wayland
Vesktop Version: 0.3.1
Tested on: AppImage & Portable Binary

The issue can be described as the web elements load for a second right before everything turns white with the console message:

```
[arRPC > bridge] web disconnected
[26747:0925/003629.417540:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
```
![Screenshot_20230925_003909](https://github.com/Vencord/Vesktop/assets/103802898/196f8988-2ab6-4fad-97a7-76397c1b2514)

<details>
Gtk-Message: 00:40:56.002: Failed to load module "appmenu-gtk-module"
[arRPC > bridge] listening on 1337
(node:27363) UnhandledPromiseRejectionWarning: ReferenceError: IS_EXTENSION is not defined
    at VencordDesktopMain:5:737
    at VencordDesktopMain:5:225
    at VencordDesktopMain:5:2229
    at VencordDesktopMain:5:225
    at VencordDesktopMain:5:2657
    at emitUnhandledRejectionWarning (node:internal/process/promises:200:15)
    at processPromiseRejections (node:internal/process/promises:296:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
(node:27363) ReferenceError: IS_EXTENSION is not defined
    at VencordDesktopMain:5:737
    at VencordDesktopMain:5:225
    at VencordDesktopMain:5:2229
    at VencordDesktopMain:5:225
    at VencordDesktopMain:5:2657
[arRPC > ipc] listening at /run/user/1000/discord-ipc-0
[arRPC > websocket] listening on 6463
[arRPC > process] started
[arRPC > bridge] web connected
Error occurred in handler for 'VencordGetUpdates': Error: No handler registered for 'VencordGetUpdates'
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:89565)
    at WebContents.emit (node:events:513:28)
[arRPC > bridge] web disconnected
[27401:0925/004101.561466:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
</details>

**Comment by MiMillieuh** [comment]
Same issue

Ublue (Silverblue) 38 Gnome Wayland

**Comment by spacebanana420** [comment]
This also happens to me. Tested on portable binary with desktop version 0.3.1

NixOS Tapir
KDE Plasma on X11

**Comment by Vendicated** [comment]
try running with any of --disable-gpu, --disable-sandbox and --disable-gpu-sandbox

**Comment by spacebanana420** [comment]
> try running with any of --disable-gpu, --disable-sandbox and --disable-gpu-sandbox

Neither worked for me sadly

**Comment by Vendicated** [comment]
fixed in 0.3.2 which will be out in ~10 mins

**Comment by OmegaRogue** [comment]
Im getting this with the current version of vesktop
